commit cb0dde63c54da62945c20171f7209f3e1a89b360
Author: Chris Friedt <cfriedt@tenstorrent.com>
Date:   Sun Nov 23 13:28:31 2025 -0500

    fdtable: add more poll-related constants
    
    Add constants ZVFS_POLLRDNORM, ZVFS_POLLWRNORM, ZVFS_POLLRDBAND, and
    ZVFS_POLLWRBAND.
    
    For more details, please see
    https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/poll.h.html
    
    Signed-off-by: Chris Friedt <cfriedt@tenstorrent.com>

diff --git a/include/zephyr/sys/fdtable.h b/include/zephyr/sys/fdtable.h
index a5c93350089..b3276510f25 100644
--- a/include/zephyr/sys/fdtable.h
+++ b/include/zephyr/sys/fdtable.h
@@ -71,6 +71,10 @@
 #define ZVFS_POLLERR  BIT(3)
 #define ZVFS_POLLHUP  BIT(4)
 #define ZVFS_POLLNVAL BIT(5)
+#define ZVFS_POLLRDNORM BIT(6)
+#define ZVFS_POLLWRNORM ZVFS_POLLOUT
+#define ZVFS_POLLRDBAND BIT(7)
+#define ZVFS_POLLWRBAND BIT(8)
 
 #ifdef __cplusplus
 extern "C" {
